pkg/virt-handler/non-root.go

```diff
@@ -124,7 +124,7 @@ func getTapDevices(vmi *v1.VirtualMachineInstance, networkBindings map[string]v1
 		_, isTapNetwork := tapNetworks[net.Name]
 		if podInterfaceName, exists := networkNameScheme[net.Name]; isTapNetwork && exists {
 			tapDevices[net.Name] = podInterfaceName
-		} else if isTapNetwork && !exists {
+		} else if isTapNetwork {
```

Comment:
maybe something like this will be more readable?
```go
	for _, net := range vmi.Spec.Networks {
		
		if _, isTapNetwork := tapNetworks[net.Name]; isTapNetwork {
				if podInterfaceName, exists := networkNameScheme[net.Name]; exists {
					tapDevices[net.Name] = podInterfaceName
				} else {
					return nil, fmt.Errorf("network %q not found in naming scheme: this should never happen", net.Name)
				}
		}
	}

```

Reply:
That would have been my initial suggestion.

Reply:
https://github.com/kubevirt/kubevirt/pull/13630#discussion_r1913019628
My comment fell into oblivion

Reply:
This whole function was removed as part of https://github.com/kubevirt/kubevirt/pull/13329/files and after rebase is removed from this PR. Therefore I'll mark this as resolved. Thanks for your suggestion and the effort of writing this code snippet :)